k8s/helmfile/tailscale-operator.yaml

```diff
@@ -0,0 +1,10 @@
+repositories:
```

Comment:
Sub-Helmfiles will unfortunately need to repeat this block: https://github.com/wmde/wbaas-deploy/blob/097a6bf415eccd37843397ab336e910d373e7f8c/k8s/helmfile/istio-control-plane.yaml#L1-L18, else it's possible to accidentally target different environments when running `make apply-env` in certain scenarios.